src/Symfony/Component/HttpFoundation/HeaderBag.php

```diff
@@ -137,10 +144,8 @@ public function get($key, $default = null, $first = true)
     public function set($key, $values, $replace = true)
     {
         $key = str_replace('_', '-', strtolower($key));
-
```

Comment:
all black lines should be added back, please don't change CS when it's unrelated, never